### PR TITLE
Created backup branch for BMR30

### PR DESCRIPTION
**Pull request by kclaudeeager**

### CODE REVIEW

File: ui/swip_animation/src/main/java/com/amalitech/swip_animation/SwipeableCardSideContents.kt [hunk 1]
```diff
@@ -0,0 +1,94 @@
+package com.amalitech.swip_animation
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
```
Comment (esthcarelle) [inline ui/swip_animation/src/main/java/com/amalitech/swip_animation/SwipeableCardSideContents.kt:7]: Can you please format the codes?

**Review by esthcarelle — CHANGES_REQUESTED** [review]

**Comment by esthcarelle** [comment]
@kclaudeeager Screenshots please

**Comment by kclaudeeager** [comment]
> @kclaudeeager Screenshots please

How do I do it? can I add a file in the commits?

**Comment by esthcarelle** [comment]
@kclaudeeager you can add an image file in the description of the PR

**Comment by kclaudeeager** [comment]
https://github.com/AmaliTech-Training-Rw/Meeting-Room-booking-App/assets/71407602/04ec92da-6c88-49dc-90b3-4a9e42013668

this is the screen recording of swiping animation

**Review by esthcarelle — APPROVED** [review]

File: ui/swip_animation/src/main/java/com/amalitech/swip_animation/SwipeableCardSideContents.kt [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    var isRightContentVisible by remember { mutableStateOf(isRightVisible) }
+    val swipeDirection = remember { mutableStateOf(SwipeDirection.None) }
+
+    Card(
+        modifier = modifier
+            .clip(RoundedCornerShape(8.dp))
+            .padding(4.dp)
+            .shadow(elevation = 4.dp)
+    ) {
+        Box(modifier = Modifier.fillMaxHeight()) {
```
Comment (bensek) [inline ui/swip_animation/src/main/java/com/amalitech/swip_animation/SwipeableCardSideContents.kt:43]: I don't see the need of this Box

File: ui/swip_animation/src/main/java/com/amalitech/swip_animation/util/SwipeDirection.kt [hunk 3]
```diff
@@ -0,0 +1,5 @@
+package com.amalitech.swip_animation.util
+
+enum class SwipeDirection {
+    None, Left, Right
```
Comment (bensek) [inline ui/swip_animation/src/main/java/com/amalitech/swip_animation/util/SwipeDirection.kt:4]: Make these Capital Letters NONE, LEFT, RIGHT

File: app/build.gradle.kts [hunk 4]
```diff
@@ -99,7 +99,11 @@ dependencies {
     implementation(project(Modules.domainOnboarding))
     implementation(project(Modules.uiOnboarding))
     implementation(project(Modules.coreUI))
+
+    implementation(project(Modules.uiSwipe))
```
Comment (bensek) [inline app/build.gradle.kts:103]: We don't need a separate module for this. Use the coreUI module and delete this you have added.

**Review by bensek — CHANGES_REQUESTED** [review]

**Review by bensek — APPROVED** [review]

**Pull request merged**